### PR TITLE
imageio and imageio-ffmpeg

### PR DESCRIPTION
**Pull request by edmondac**
For INC1010122

`imageio-ffmpeg-0.3.0-foss-2019b-Python-3.7.4.eb imageio-2.6.1-foss-2019b-Python-3.7.4.eb`
* [ ] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-sandybridge
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/i/imageio/imageio-2.6.1-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+sources = [SOURCE_TAR_GZ]
+checksums = ['f44eb231b9df485874f2ffd22dfd0c3c711e7de076516b9374edea5c65bc67ae']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Pillow', '6.2.1'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+download_dep_fail = True
```
Comment (orbsmiv) [inline easybuild/easyconfigs/i/imageio/imageio-2.6.1-foss-2019b-Python-3.7.4.eb]: Should this also have the `sanity_pip_check = True` line?

Comment (orbsmiv) [inline easybuild/easyconfigs/i/imageio/imageio-2.6.1-foss-2019b-Python-3.7.4.eb]: Should this also have the `sanity_pip_check = True` line?

Comment (edmondac) [reply easybuild/easyconfigs/i/imageio/imageio-2.6.1-foss-2019b-Python-3.7.4.eb]: Done

**Comment by orbsmiv** [comment]
Built on all requested platforms

**Pull request merged**